pkg/downloader/manager.go

```diff
@@ -206,6 +206,7 @@ func (m *Manager) downloadAll(deps []*chartutil.Dependency) error {
 	}
 
 	destPath := filepath.Join(m.ChartPath, "charts")
+	tmpPath := filepath.Join(m.ChartPath, "tmpcharts")
```

Comment:
TempDir creates the directory, I use rename on existing charts dir so I don't need to move all its contents.

I think concurrent `dep up` calls would break anyway as we move old charts dir and if one of the commands fail to download we would replace existing state with new chart - possibly during other `dep up` download, which would probably fail miserably. Making tmp dir names random would not help here.